kytos/core/api_server.py

```diff
@@ -59,6 +59,24 @@ def register_rest_endpoint(self, url, function, methods):
             url = url[1:]
         self._start_endpoint(f'/kytos/{url}', function, methods=methods)
 
+    def remove_rest_endpoints(self, napp_id):
```

Comment:
`s/remove_rest_endpoint/remove_napp_rest_endpoints`
(This method is designed to remove endpoints related to `napp_id`)